compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaRegexpTypeSymbol.java

```diff
@@ -19,26 +19,44 @@
 
 import io.ballerina.compiler.api.SymbolTransformer;
 import io.ballerina.compiler.api.SymbolVisitor;
+import io.ballerina.compiler.api.impl.SymbolFactory;
+import io.ballerina.compiler.api.symbols.ModuleSymbol;
 import io.ballerina.compiler.api.symbols.RegexpTypeSymbol;
 import io.ballerina.compiler.api.symbols.TypeDescKind;
+import org.wso2.ballerinalang.compiler.semantics.model.SymbolTable;
 import org.wso2.ballerinalang.compiler.semantics.model.types.BRegexpType;
 import org.wso2.ballerinalang.compiler.util.CompilerContext;
 import org.wso2.ballerinalang.compiler.util.Names;
 
+import java.util.Optional;
+
 /**
  * Represents an RegExp type descriptor.
  *
  * @since 2201.3.0
  */
 public class BallerinaRegexpTypeSymbol extends AbstractTypeSymbol implements RegexpTypeSymbol {
 
+    private ModuleSymbol module;
+
     public BallerinaRegexpTypeSymbol(CompilerContext context, BRegexpType regexpType) {
         super(context, TypeDescKind.REGEXP, regexpType);
     }
 
     @Override
     public String signature() {
-        return "regexp:" + Names.REGEXP;
+        return "regexp:" + Names.STRING_REGEXP;
```

Comment:
Can we do a signature assertion to cover this?

Reply:
I'll add.